specification/mediaservices/resource-manager/Microsoft.Media/Streaming/stable/2022-11-01/streamingservice.json

```diff
@@ -2440,6 +2596,209 @@
       },
       "description": "The status of an async operation.",
       "type": "object"
+    },
+    "LiveEventIngestInterruption": {
+      "properties": {
+        "begin": {
+          "description": "UTC time of interruption start, encoder disconnected.",
+          "type": "string"
+        },
+        "end": {
+          "description": "UTC time of interruption end, encoder re-connected.",
+          "type": "string"
+        },
+        "durationInSec": {
+          "description": "Duration of interruption in seconds.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "reason": {
+          "description": "Interruption reason.",
+          "type": "string"
+        }
+      },
+      "description": "The live event ingest interruption data.",
+      "type": "object"
+    },
+    "LiveEventIngestion": {
+      "properties": {
+        "streamName": {
+          "description": "Ingestion stream name.",
+          "type": "string"
+        },
+        "beginTime": {
+          "description": "Ingestion begin time in UTC.",
+          "type": "string"
+        },
+        "endTime": {
+          "description": "Ingestion end time in UTC. Empty if it's not stopped yet.",
+          "type": "string"
+        },
+        "endReason": {
+          "description": "End reason.",
+          "type": "string"
+        },
+        "ingestInterruptions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LiveEventIngestInterruption"
+          },
+          "x-ms-identifiers": [],
+          "description": "The list of ingest interruptions."
+        }
+      },
+      "description": "The live event ingestion telemetry data.",
+      "type": "object"
+    },
+    "LiveEventTrackStatus": {
+      "properties": {
+        "trackId": {
+          "description": "Track Id.",
+          "type": "string"
+        },
+        "expectedBitrate": {
+          "description": "Expected bitrate for this track.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "incomingBitrate": {
+          "description": "Average incoming bitrate for last 20 seconds when live event is running.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "ingestDrift": {
+          "description": "Current ingest drift value in seconds for last 1 min.",
+          "type": "string"
+        }
+      },
+      "description": "The live event track status.",
+      "type": "object"
+    },
+    "LiveEventStatus": {
+      "properties": {
+        "state": {
+          "description": "Current state of the live event. Running | Stopped.",
+          "type": "string"
+        },
+        "healthStatus": {
+          "description": "The health status over the last 20 seconds.",
+          "type": "string"
+        },
+        "healthDescription": {
```

Comment:
I am not sure this name works here. isn't this a list of issues?

Reply:
I'll let Daniel to answer this.

Reply:
I'll update this to `healthDescriptions`